api/server/router/network/network.go

```diff
@@ -31,8 +31,7 @@ func (r *networkRouter) initRoutes() {
 	r.routes = []router.Route{
 		// GET
 		router.NewGetRoute("/networks", r.getNetworksList),
-		router.NewGetRoute("/networks/", r.getNetworksList),
```

Comment:
This route was added on purpose; see https://github.com/docker/docker/pull/24634